js/modules/k6/experimental/streams/readable_streams.go

```diff
@@ -62,9 +62,10 @@ func (stream *ReadableStream) Cancel(reason goja.Value) *goja.Promise {
 // GetReader implements the [getReader] operation.
 //
 // [getReader]: https://streams.spec.whatwg.org/#rs-get-reader
-func (stream *ReadableStream) GetReader(options *goja.Object) goja.Value {
+func (stream *ReadableStream) GetReader(options *sobek.Object) sobek.Value {
 	// 1. If options["mode"] does not exist, return ? AcquireReadableStreamDefaultReader(this).
-	if options == nil || common.IsNullish(options) || options.Get("mode") == nil || goja.IsUndefined(options.Get("mode")) {
+	if options == nil || common.IsNullish(options) ||
+		options.Get("mode") == nil || sobek.IsUndefined(options.Get("mode")) {
```

Comment:
This unfortunately becomes 121 symbols and also both cases hit the "is interface nil?" trap so ... yeah. This is the best I can do without having to add a lot mroe code.